libsession-util/src/main/java/network/loki/messenger/libsession_util/util/BlindKeyAPI.kt

```diff
@@ -0,0 +1,10 @@
+package network.loki.messenger.libsession_util.util
+
+object BlindKeyAPI {
+    init {
+        System.loadLibrary("session_util")
```

Comment:
would perhaps be ideal to just have one `by lazy` somewhere that we reference instead of loading this thing for the 8th time.

IDK if it's optimised away sometimes/always/never, but at least when we loaded a lib way to many times, it crashed on some devices.